examples/linear_model.ipynb

```diff
@@ -0,0 +1,306 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from prog_models import LinearModel"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll define some features of ThrownObject such that it models after a LinearModel. Remember at the top of the Jupyter Notebook, there were a few properties that all Linear Models must define? In the next step, we'll define our inputs, states, outputs, and events, along with the $A$, $C$, $E$, and $F$ values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(LinearModel):\n",
+    "    events = ['impact']\n",
+    "    inputs = []  \n",
+    "    states = ['x', 'v']\n",
+    "    outputs = ['x']\n",
+    "    \n",
+    "    A = np.array([[0, 1], [0, 0]])\n",
+    "    C = np.array([[1, 0]])\n",
+    "    E = np.array([[0], [-9.81]])\n",
+    "    F = None"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll define some default parameters for our ThrownObject model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(ThrownObject):  # Continue the ThrownObject class\n",
+    "    default_parameters = {\n",
+    "        'thrower_height': 1.83,\n",
+    "        'throwing_speed': 40,\n",
+    "        'g': -9.81\n",
+    "    }"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following cells, we'll define some necessary class functions such that we are able to perform prognostics on the model!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `initialize()` function sets the states to their corresponding values. E.g., we have defined that our `x` value for our ThrownObject model to represent position in space, therefore, our initial value for the position in space would be the thrower_height parameter!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(ThrownObject):\n",
+    "    def initialize(self, u=None, z=None):\n",
+    "        return self.StateContainer({\n",
+    "            'x': self.parameters['thrower_height'],\n",
+    "            'v': self.parameters['throwing_speed']\n",
+    "            })"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For our `threshold_met()`, we have defined the function to return a boolean function when our thrown object has met the initial state of 'impact', or in other words, has hit the ground. This occurs when our height is less than or equal to 0, and our speed is less than 0!"
```

Comment:
I think the two different events are getting a bit mixed up here. Let's try rewriting it to clearly describe the two different events we are considering: 'impact' (i.e. when height is less than zero and it has hit the ground), and 'falling' (i.e. when velocity is less than 0)